frontend/src/pages/modelRegistry/screens/routeUtils.ts

```diff
@@ -1,8 +1,21 @@
 export const registeredModelUrl = (rmId?: string, preferredModelRegistry?: string): string =>
   `/modelRegistry/${preferredModelRegistry}/registeredModels/${rmId}`;
 
+export const modelVersionListUrl = (rmId?: string, preferredModelRegistry?: string): string =>
+  `${registeredModelUrl(rmId, preferredModelRegistry)}/versions`;
+
 export const modelVersionUrl = (
   mvId: string,
   rmId?: string,
   preferredModelRegistry?: string,
 ): string => `${registeredModelUrl(rmId, preferredModelRegistry)}/versions/${mvId}`;
+
+export const modelVersionArchiveUrl = (rmId?: string, preferredModelRegistry?: string): string =>
+  `/modelRegistry/${preferredModelRegistry}/registeredModels/${rmId}/versions/archive`;
+
+export const modelVersionArchiveDetailsUrl = (
+  mvId: string,
+  rmId?: string,
+  preferredModelRegistry?: string,
+): string =>
+  `/modelRegistry/${preferredModelRegistry}/registeredModels/${rmId}/versions/archive/${mvId}`;
```

Comment:
I think this route can be changed to `archived` instead of `archive`, since it's already archived?

Reply:
`archive` sounds better to me, as it denotes a list of archived items. @lucferbux, any thoughts on this?

Reply:
That's something that we'll need to run down with the UX team for microcopy, IMO if it were `archivedModels` i would go with the past but given that it's a state `archive` sounds better.